index.html

```diff
@@ -1441,6 +1446,21 @@ <h2>Security Considerations</h2>
           Security Considerations</a>
         </li>
       </ul>
+      <p>
+        Security considerations for features introduced in this specification can be located in their individual
+        sections:
+      </p>
+      <ul>
+        <li>
+          <a href="#external-content"></a>
```

Comment:
My lack of recall regarding respec is showing here, but: are these intentionally empty `<a>` elements?

Reply:
respec fills them in with the appropriate section title